schema.sql/schema

```diff
@@ -0,0 +1,57 @@
+-- Create Database
+CREATE DATABASE catalog_of_things;
+DROP IF EXISTS music_albums, genres, books, labels, sources, authors, games;
+
+
+CREATE TABLE labels (
+  id SERIAL PRIMARY KEY,
+  label_title TEXT,
+  label_color TEXT
+);
+
+CREATE TABLE books (
+  id SERIAL PRIMARY KEY,
+  title TEXT,
+  publisher TEXT,
+  cover_state TEXT,
+  publish_date DATE,
+  archived BOOLEAN,
+  label_id INTEGER,
+  FOREIGN KEY (label_id) REFERENCES labels(id)
+);
+
+
+CREATE TABLE authors (
+  id SERIAL PRIMARY KEY,
+  first_name VARCHAR(100) NOT NULL,
+  last_name VARCHAR(100) NOT NULL,
+  PRIMARY key (id)
+)
+
+CREATE TABLE games (
+  games_id INT,
```

Comment:
- [ ] As clearly pointed out by the previous reviewer, please kindly include an `items` table that has a `belongs to` relationship with four other tables.